openquakeplatform/static/exposure/exposure_controller.js

```diff
@@ -54,18 +54,7 @@ var showErrorDialog = function(message, options) {
 };
 
 var downloadFractions = function(srId) {
-    $.ajax({
-        type: 'get',
-        url: 'export_fractions_by_study_region_id?sr_id='+srId,
-        success: function(data, textStatus, jqXHR) {
-            if (navigator.appName != 'Microsoft Internet Explorer') {
-                window.open('data:text/csv;charset=utf-8,' + escape(data));
-            } else {
-                var popup = window.open('','csv','');
-                popup.document.body.innerHTML = '<pre>' + data + '</pre>';
-            }
-        }
-    });
+    window.open('export_fractions_by_study_region_id?sr_id='+srId);
```

Comment:
Add a comment about the change and what it implies about empty new tabs (and the fact that the right solution is using the StreamingResponse)

Reply:
Fixed here: Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/a590f2ef001b310fd7446306c1ff997a0d186113

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/b138645df415e30139dd5747d6b2a742f1876d97